content/posts/container-structure-test.md

```diff
@@ -7,9 +7,9 @@ categories: ["containers", "testing"]
 featuredImage: "/images/containers.jpg"
 ---
 
-It is no secret that when we are writing software, tests are a critical component to ensuring that the code does what we say it does. It is so critical that most languages come with testing frameworks. JavaScript has testing frameworks such as [mocha](https://mochajs.org/) and [jasmine](https://jasmine.github.io/). Go ships with its own testing capabilities provided by the [testing package](https://golang.org/pkg/testing/). And while writing tests in these languages is an accepted standard practice, all too often, we forget that there's more to getting an application onto production than the app itself.
+It is no secret that when we are writing software, tests are a critical component to ensure the code does what we say it does. It is so critical that most languages come with testing frameworks. JavaScript has testing frameworks such as [mocha](https://mochajs.org/) and [jasmine](https://jasmine.github.io/). Go ships with its own testing capabilities provided by the [testing package](https://golang.org/pkg/testing/). And while writing tests in these languages is an accepted standard practice, all too often we forget that there's more to getting an application onto production than the app itself.
 
-Dockerfiles play a big part in how we ship software at Plex. We use them heavily in our build pipelines to run [containerized jobs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases?view=azure-devops) as well as the distribution mechanism to get the software onto our Kubernetes clusters.  We like to say, "If it's code, we can test it." and that is no exception when it comes to writing our Dockerfiles.
+Dockerfiles play a big part in how we ship software at Plex. We use them heavily in our build pipelines to run [containerized jobs](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases?view=azure-devops) as well as the distribution mechanism to get the software onto our Kubernetes clusters.  We like to say, "If it's code, we can test it," and and that is no exception when it comes to writing our Dockerfiles.
```

Comment:
It's currently reading as "we can test it," and and  that is no exception.  Remove the second "and" and put the comma after the end quote

Reply:
I fixed the `and and`, that definitely screamed at me in my editor. For the comma, I believe it goes inside the quotes. Jerry had pointed this out in his email, and reading through some grammar rules (https://www.grammarbook.com/punctuation/quotes.asp) it seems correct now.